sdk/dotnet/AppRole/AuthBackendLogin.cs

```diff
@@ -127,12 +127,12 @@ public partial class AuthBackendLogin : Pulumi.CustomResource
         /// <param name="args">The arguments used to populate this resource's properties</param>
         /// <param name="options">A bag of options that control this resource's behavior</param>
         public AuthBackendLogin(string name, AuthBackendLoginArgs args, CustomResourceOptions? options = null)
-            : base("vault:appRole/authBackendLogin:AuthBackendLogin", name, args ?? new AuthBackendLoginArgs(), MakeResourceOptions(options, ""))
+            : base("vault:approle/authBackendLogin:AuthBackendLogin", name, args ?? new AuthBackendLoginArgs(), MakeResourceOptions(options, ""))
```

Comment:
Is this right? Do we really want to be changing the type here?

Reply:
I am going to defer this question to @mikhailshilkov

Reply:
This is not a .NET thing - it changed in all languages an in schema. And it does look suspicious - do you know why this happened?